index.js

```diff
@@ -38,6 +38,7 @@ function requestbody(opts) {
   opts.jsonLimit = 'jsonLimit' in opts ? opts.jsonLimit : '1mb';
   opts.formLimit = 'formLimit' in opts ? opts.formLimit : '56kb';
   opts.formidable = 'formidable' in opts ? opts.formidable : {};
+  opts.textLimit = 'textLimit' in opts ? opts.formLimit : '56kb';
```

Comment:
Shouldn't formLimit be textLimit here?

Could you add a test for parameter parsing?

Thanks!